frontends/p4/fromv1.0/converters.cpp

```diff
@@ -508,10 +508,10 @@ const IR::Statement *PrimitiveConverter::cvtPrimitive(ProgramStructure *structur
     return nullptr;
 }
 
-vector<const IR::Expression *>
+safe_vector<const IR::Expression *>
```

Comment:
It won't be easy for people to remember to use safe_vector.

Reply:
That's true. I'd argue that remembering `vector` vs `std::vector` is even more difficult, though. =)

It's probably actually a better bet to just use the built-in debugging modes for the various C++ standard library implementations (i.e., define `_GLIBCXX_DEBUG` and `_LIBCPP_DEBUG` for debug builds). That way we get coverage everywhere, and nobody has to remember anything special. I'll run some benchmarks, and if it doesn't result in a massive slowdown I'll open a PR to just use that approach and completely remove `safe_vector`.